internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
+	"github.com/google/go-github/v68/github"
+	"github.com/labstack/echo/v4"
```

Comment:
Ensure that the `ctx` passed to `GithubController` is not nil. If it is nil, it could lead to unexpected behavior when using this context for API requests or other operations that rely on the context being valid.